### PR TITLE
Linter fixes

### PR DESCRIPTION
**Pull request by GarethLewin**
Your choice on this, required if you want linting to run in CI

clean with `gometalinter ./... --vendor --disable gocyclo --disable gas
--disable goconst` the errors in `gas` are mainly because you are
shelling out to a variable (security issue, but required for retool to
work) and goconst doesn't like the reuse of constants like `"clean"` and
`"add"`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+go_import_path: github.com/twitchtv/retool
+
 go:
+- 1.7.5
 - 1.8
-go_import_path: github.com/twitchtv/retool
+- tip
+
 script:
+- go get github.com/alecthomas/gometalinter
```
Comment (spenczar) [inline .travis.yml]: Instead of installing gometalinter here, let's install it with `retool` itself.

Comment (GarethLewin) [inline .travis.yml]: I thought of that, but not sure if that is circular? Also with gml vendoring it's own tools is that easy?

Alternatively we can use retool on the individual linters and not use gml?

**Comment by GarethLewin** [comment]
Updated. I recommend squash merging this.

**Comment by spenczar** [comment]
Thanks a lot!

**Pull request merged**